### DISCUSSION TITLE
Let issue authors reopen their own issues

### DISCUSSION THREAD

**Discussion by kittykat** [community/community#5084] [Projects and Issues]
It would be really useful to be able to let issue authors who are not members of the project to reopen their own issues in cases where the issue cannot be reproduced or does not have enough information in it and there is no immediate response from the author. Instructing them to open a new issue creates duplicates which make searching the issue history more difficult.

See also [previous discussion](https://github.com/isaacs/github/issues/583)

**Comment by Marcono1234** [comment]
That could also (partially) solve issues with "stale" bots and actions such as https://github.com/actions/stale/issues/345.

**Comment by NitinIsFit** [comment]
Then you will have to make an another option for admins "restrict author to reopen this issue" 🤦‍♂️

**Comment by MrSarno** [comment]
I would be concerned about the possibility of abuse by people who want to see something changed, even if it's either not feasible to implement the desired change, or the existing functionality is working as intended. I think it makes sense that arbitrary individuals can't impose open issues on organisations maintaining software in cases where the maintainers want those issues to be closed.

If the issue hasn't been _locked_, it should be possible to politely request that it be re-opened.

**Comment by 2rs2ts** [comment]
In my experience, organizations that use stale bots never have any interest in reopening issues that have been closed because they're stale, since it's an automated way to get out of doing work, even if the work is highly requested and the issue is still plaguing users. Allowing the issue creator to reopen as a special exception will probably not result in their request being addressed; however, it will force the project maintainers to get their hands dirty and _lock_ the issue (something which is usually seen as very rude and unwelcoming.) Whether or not you think the platform supports this depends on your personal values and how much you anticipate having to deal with bad actors who keep reopening their issues because they want to force you to lock them (which I think is going to be a pretty rare occurrence, tbh.)

It's worth keeping in mind that users will just create duplicate issues if something got closed and they can't reopen it (lock or no lock.) I have seen it happen in the Kubernetes org a few times at least. Allowing them to reopen might be the lesser of two evils.

**Comment by marcindulak** [comment]
Collected a few more discussions asking for that feature in https://github.com/orgs/community/discussions/182939.
Consider going to all of them an upvoting them.
